jenkins/gradle/gradle-check-flaky-test-issue-creation.jenkinsfile

```diff
@@ -14,7 +14,14 @@ lib = library(identifier: 'jenkins@6.9.2', retriever: modernSCM([
 
 
 pipeline {
-    agent { label 'Jenkins-Agent-Ubuntu2004-X64-M58xlarge-Single-Host' }
+    agent {
+        docker {
+            label 'Jenkins-Agent-AL2023-X64-M54xlarge-Docker-Host'
+            image 'opensearchstaging/ci-runner:ci-runner-al2-opensearch-build-v1'
```

Comment:
Maybe you can use the rockylinux8 one?
Does the AL2 once has the correct curl version?

Reply:
Yes! It does. 
```
curl --version
curl 8.3.0 (aarch64-koji-linux-gnu) libcurl/8.3.0 OpenSSL/1.0.2k-fips zlib/1.2.7 libidn2/2.3.0 libpsl/0.21.5 (+libidn2/2.3.0) libssh2/1.4.3 nghttp2/1.41.0 OpenLDAP/2.4.44
Release-Date: 2023-09-13
Protocols: dict file ftp ftps gopher gophers http https imap imaps ldap ldaps mqtt pop3 pop3s rtsp scp sftp smb smbs smtp smtps telnet tftp ws wss
Features: alt-svc AsynchDNS GSS-API HSTS HTTP2 HTTPS-proxy IDN IPv6 Kerberos Largefile libz NTLM NTLM_WB PSL SPNEGO SSL threadsafe UnixSockets
```